File: templates/Features/SuspendAndResumeVB/Services/OnBackgroundEnteringEventArgs.vb
Namespace Services
    Public Class OnBackgroundEnteringEventArgs
        Inherits EventArgs

        Public Property SuspensionState As SuspensionState
        
        Public Property Target As Type

        Public Sub New(suspensionState As SuspensionState, target As Type)
            SuspensionState = suspensionState
            Target = target
        End Sub
      End Class
End Namespace 

File: templates/Uwp/Features/SuspendAndResume/Services/OnBackgroundEnteringEventArgs.cs
using System;

namespace Param_RootNamespace.Services
{
    public class OnBackgroundEnteringEventArgs : EventArgs
    {
        public SuspensionState SuspensionState { get; set; }

        public Type Target { get; private set; }

        public OnBackgroundEnteringEventArgs(SuspensionState suspensionState, Type target)
        {
            SuspensionState = suspensionState;
            Target = target;
        }
    }
}
